src/app/core/config/config-fix.ts

```diff
@@ -86,7 +86,7 @@ export const defaultJsonConfig = {
     "config": {
       "widgets": [
         {
-          "component": "DashboardShortcutWidget",
+          "component": "ShortcutDashboard",
```

Comment:
This will require a (simple) migration.

Reply:
"DashboardShortcutWidget" is registered as an alias in the new module. Or do we prefer to run a migration?

Reply:
Ah right, but generally the migration is easy to achieve with search and replace so we might as well do it now.

Reply:
Maybe a point to also finally do this for the other aliases.

Reply:
:+1: